contracts/src/bridge/Inbox.sol

```diff
@@ -112,6 +112,7 @@ contract Inbox is DelegateCallAware, PausableUpgradeable, IInbox {
         }
         allowListEnabled = false;
         bridge = _bridge;
+        _pause();
```

Comment:
I don't think this is necessary, because until stage3, the inbox isn't an authorized inbox for the bridge regardless.

Reply:
thats true, will revert the commit